packages/odyssey-react-mui/src/OdysseyProvider.tsx

```diff
@@ -60,6 +60,25 @@ const OdysseyProviderInner = <SupportedLanguages extends string>({
 }: OdysseyProviderProps<SupportedLanguages>) => {
   const { contrastMode } = useContrastContext();
 
+  const memoizedThemeProps = useMemo(
```

Comment:
Sorry, i just meant the override styles themselves. Something like this:
```js
const memoizedThemeProps = useMemo(
    () => ({
        ...themeOverride,
        odysseyContrastMode: contrastMode,
    }),
    [
      themeOverride,
      contrastMode,
    ],
  );```

Reply:
But, your solution is probably fine too

Reply:
got it! I like your suggestion, its easier to read, IMO